aicsimageio/readers/lif_reader.py

```diff
@@ -605,6 +605,22 @@ def _construct_mosaic_xarray(self, data: types.ArrayLike) -> xr.DataArray:
                 DimensionNames.SpatialX,
             ]
         }
+
+        # Add expanded Y and X coords
+        scale_x, scale_y, _, _ = selected_scene.info["scale"]
+        if scale_y is not None:
+            coords[DimensionNames.SpatialY] = np.arange(
+                0,
+                stitched.shape[-2] * scale_y,
+                scale_y,
+            )
+        if scale_x is not None:
+            coords[DimensionNames.SpatialX] = np.arange(
+                0,
+                stitched.shape[-1] * scale_x,
+                scale_x,
+            )
```

Comment:
Do we have any tests to check against this? Would that be too cumbersome?

Reply:
I will add some. Shouldn't be too hard at all :)